webapp/config.py

```diff
@@ -50,4 +50,5 @@ class ProdConfig(ConfigBase):
     'Prod': ProdConfig()
 }
 
-ENVAR_FINTECH_CONFIG = 'FINTECH_CONFIG'
\ No newline at end of file
+ENVAR_FINTECH_CONFIG = 'FINTECH_CONFIG'
+os.environ['FINTECH_CONFIG'] = 'Dev'
```

Comment:
@juzar4u @harisahmed11 
why have you done this? Won't this override the original environment variable. And setting an environment variable inside code, defeats the purpose of having an environment variable in the first place !